notesh/main.py

```diff
@@ -27,12 +28,17 @@ class NoteApp(App):
         Binding("ctrl+c", "quit", "Quit"),
     ]
 
+    DEFAULT_FILE = os.environ.get(
+        "NOTESH_FILE",
+        str(Path(os.environ.get("XDG_DATA_HOME", Path("~/.local/share").expanduser())) / "notesh" / "notes.json"),
+    )
+
```

Comment:
Project currently should work for both Linux and Windows so this part should be universal. I would put this line outside the class and do like:
```python
if os.name == 'nt':  # Windows
    default_dir = Path(os.getenv('APPDATA')) / 'notesh'
else:  # Unix/Linux/Mac
    default_dir = Path(os.getenv("XDG_DATA_HOME", Path("~/.local/share").expanduser())) / 'notesh'

DEFAULT_FILE = Path(os.environ.get("NOTESH_FILE", default_dir / "notes.json"))
```

Reply:
I don't have a Windows box here to test though.